.github/workflows/manifest.yml

```diff
@@ -20,19 +18,25 @@ jobs:
           fetch-depth: 0
           persist-credentials: false
 
+      - name: west setup
+        env:
+          BASE_REF: ${{ github.base_ref }}
+        working-directory: zephyrproject/zephyr
```

Comment:
This dir is empty in main. Should this point to ncs/nrf ?

Reply:
fixed